.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,47 @@
+---
+name: 🐞 Bug Report
+about: Something went awry and you'd like to tell us about it.
+
+---
+
+<!--
+  ⚡️ katchow! We 💛 issues.
+
+  Please - do not - remove this template.
+  Please - do not - skip or remove parts of this template.
+  Or your issue may be closed.
+
+  👉🏽 Need help or tech support? Please don't open an issue!
+  Head to https://gitter.im/rollup/rollup or https://stackoverflow.com/questions/tagged/rollupjs
+
+  ❤️ Rollup? Please consider supporting our collective:
+  👉 https://opencollective.com/rollup/donate
+-->
+
+- Rollup Version:
+- Operating System (or Browser):
+- Node Version:
+
+### How Do We Reproduce?
+
+<!--
+  Issues without minimal reproductions will be closed! Please provide one by:
+  1. Work to isolate the problem and provide the exact steps in this issue, or
+  2. Provide a minimal repository link (Read https://git.io/fNzHA for instructions).
+     These may take more time to triage than the other options.
+-->
+
+
+### Expected Behavior
+
+
+### Actual Behavior
+
+
+<!--
+  Most issues can be expressed or demonstrated through the REPL or a repository.
```

Comment:
This does not really apply here as you do not have rollup-plugin-node-resolve in the REPL (after all, the REPL runs in the browser). Maybe we just change the first line to

```
<!--
  Most issues can be expressed or demonstrated through a repository.
  However ...
```

Reply:
For clarity maybe `Most issues can be expressed or demonstrated through a reproduction repository. However ...` ?

Does it sound good in English? I don't know him very well, correct me if I suggested nonsense :)